css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  justify-content: center;
+  background-color: #1996da;
+  color: white;
+  font-family: lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.container {
+  margin: auto;
+  max-width: 1000px;
+}
+
+.row {
+  display: flex;
+}
+
+.columns {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+.image-container {
+  background-color: white;
+  height: 10rem;
+  width: 10rem;
+  border-radius: 50%;
+  justify-content: center;
+  text-align: center;
+  margin: auto;
+}
+
+.image-container img {
+  background-color: white;
+  text-align: center;
+  justify-content: center;
+  height: 5rem;
+  width: 5rem;
+  margin-top: 2.5rem;
+  flex-basis: calc(100%/3);
```

Comment:
I believe you can remove a lot of these properties. Try removing them one-by-one to see what actually has an effect.